src/pages/production/planning/page.tsx

```
@@ -0,0 +1,342 @@
+
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '@/shared/ui';
```

ESLint

[error] '/home/jailuser/git/src/shared/ui/index.ts' imported multiple times.

(import/no-duplicates)